toonz/sources/translations/german/tnztools.ts

```diff
@@ -0,0 +1,1001 @@
+        <translation>Mitte［Y］ sperren</translation>
+    </message>
+    <message>
+        <source>Lock Position E/W</source>
+        <translation>Position ［X］ sperren</translation>
+    </message>
+    <message>
+        <source>Lock Position N/S</source>
+        <translation>Position［Y］ sperren</translation>
+    </message>
+    <message>
+        <source>Lock Rotation</source>
+        <translation>Rotation sperren</translation>
+    </message>
+    <message>
+        <source>Lock Shear H</source>
+        <translation>Scherung H sperren</translation>
+    </message>
+    <message>
+        <source>Lock Shear V</source>
+        <translation>Scherung V sperren</translation>
+    </message>
+    <message>
+        <source>Lock Scale H</source>
+        <translation>Maßstab H sperren</translation>
+    </message>
+    <message>
+        <source>Lock Scale V</source>
+        <translation>Maßstab V sperren</translation>
+    </message>
+    <message>
+        <source>Lock Global Scale</source>
+        <translation>Globalen Maßstab sperren</translation>
+    </message>
+    <message>
+        <source>E/W and N/S Positions</source>
+        <translation>E/W und N/S Positionen</translation>
+    </message>
+    <message>
+        <source>Z Position</source>
+        <translation>Z Position</translation>
+    </message>
+    <message>
+        <source>SO</source>
+        <translation>SO</translation>
+    </message>
+    <message>
+        <source>Rotation</source>
+        <translation>Rotation</translation>
+    </message>
+    <message>
+        <source>Global Scale</source>
+        <translation>Globalen Maßstab sperren</translation>
+    </message>
+    <message>
+        <source>Horizontal and Vertical Scale</source>
+        <translation>Horizontaler und vertikaler Maßstab</translation>
+    </message>
+    <message>
+        <source>Shear</source>
+        <translation>Scheren</translation>
+    </message>
+    <message>
+        <source>Center Position</source>
+        <translation>Mitte-Position</translation>
+    </message>
+    <message>
+        <source>Active Axis</source>
+        <translation>Aktive Axe</translation>
+    </message>
+</context>
+<context>
+    <name>EraserTool</name>
+    <message>
+        <source>Size:</source>
+        <translation>Größe:</translation>
+    </message>
+    <message>
+        <source>Hardness:</source>
+        <translation>Härte:</translation>
+    </message>
+    <message>
+        <source>Type:</source>
+        <translation>Typ:</translation>
+    </message>
+    <message>
+        <source>Mode:</source>
+        <translation>Modus:</translation>
+    </message>
+    <message>
+        <source>Selective</source>
+        <translation>Selektiv</translation>
+    </message>
+    <message>
+        <source>Invert</source>
+        <translation>Invertieren</translation>
+    </message>
+    <message>
+        <source>Frame Range</source>
+        <translation>Frame-Reichweite</translation>
```

Comment:
Range may be a "range from x to y", so "Frame-Bereich" would be clearer than "Frame-Reichweite" (What's that? Frame scope?)